tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "module": "es6",
+    "moduleResolution": "node",
+    "target": "es5",
+    "newLine": "LF",
+    "declaration": true,
+    "strictNullChecks": true,
+    "inlineSourceMap": true,
+    "inlineSources": true,
+    "outDir": "ts-dist"
+  },
+  "include": ["**/*"]
```

Comment:
that include will cause issues, in general you really want this to only be entry points, but this without an exclude will see ts-dist and node_modules